MapboxCoreNavigation/RouteController.swift

```diff
@@ -685,6 +684,12 @@ extension RouteController: CLLocationManagerDelegate {
             if secondsToEndOfStep <= RouteControllerHighAlertInterval {
                 alertLevel = .high
             }
+        } else if let _ = outletRoadClasses?.contains(.motorway),
```

Comment:
It'd be nice if we could filter out on/off ramps. These should have the normal alert level progression. Perhaps we can look if the current step has a name/dest/ref since I believe trunks do not.